src/parser/element.js

```diff
@@ -91,16 +90,12 @@ class Element {
       debug('Element %s is not allowed within %j', qname, this.nsName);
     }
     
-    if (ElementType) {
```

Comment:
is ElementType expected to be returned always ?

Reply:
As far as I understood the code, the function `getElementType` in `typeRegistry.js` never returns `null` or `undefined`. So there is no need to check for it.